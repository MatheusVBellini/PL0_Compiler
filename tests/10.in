BEGIN
  x := (10;
END.
