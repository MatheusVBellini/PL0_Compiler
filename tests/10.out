2:11 error: Missing right parenthesis `)` before this token

Compilation failed, 1 errors
